Resources/Prototypes/_Mono/Entities/Clothing/Back/backpacks.yml

```
@@ -0,0 +1,28 @@
+- type: entity
+  parent: ClothingBackpack
+  id: ClothingBackpackPerogee
+  name: perogee-pattern backpack
+  description: Despite the name and looks, its just a normal backpack. Dissapointing.
+  components:
+  - type: Sprite
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)